account-kit/smart-contracts/src/ma-v2/client/client.ts

```diff
@@ -25,13 +35,34 @@ export type CreateSMAV2AccountClientParams<
     SmartAccountClientConfig<TTransport, TChain>,
     "transport" | "account" | "chain"
   >;
+export type CreateSMAV2AlchemyAccountClientParams<
+  TTransport extends Transport = Transport,
+  TChain extends Chain = Chain,
+  TSigner extends SmartAccountSigner = SmartAccountSigner
+> = Omit<
+  CreateSMAV2AccountClientParams<TTransport, TChain, TSigner>,
+  "transport"
+> &
+  Omit<
+    AlchemySmartAccountClientConfig<TChain, LightAccount<TSigner>>,
+    "account"
+  > & { paymasterAndData?: never };
```

Comment:
📓 That without this this test fails
![image](https://github.com/user-attachments/assets/2165692d-a90c-42d3-b4e8-7cdb1ae10738)

Reply:
oh nice, I think we also need to add `dummyPaymasterAndData?: never` here as well